packages/core/src/styles/safari-focus-outline.ts

```diff
@@ -4,6 +4,17 @@ type OutlineFn = (
   theme: Theme,
   color?: keyof Theme['colors'],
   borderRadius?: string,
+  borderStyle?:
```

Comment:
Could we use CSSProperties['borderStyle'] instead of?

Reply:
Done